src/components/calcite-input-date-picker/calcite-input-date-picker.e2e.ts

```diff
@@ -85,6 +88,11 @@ describe("calcite-input-date-picker", () => {
     await input.press("0");
     await page.waitForChanges();
     expect(changedEvent).toHaveReceivedEventTimes(4);
+    const element = await page.find("calcite-input-date-picker");
+    element.setProperty("end", "2020-03-05");
+    await page.waitForChanges();
+    expect(await element.getProperty("start")).toBe("2020-03-07");
+    expect(await element.getProperty("startAsDate")).toBeDefined();
```

Comment:
Can you assert the type instead of a non-undefined value?

Reply:
I tried but it looks like the e2e tests don't actually return a date.